# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../xf86-video-ati/udev.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -Nur xf86-video-ati-6.13.1-orig/src/drmmode_display.c xf86-video-ati-6.13.1/src/drmmode_display.c
--- xf86-video-ati-6.13.1-orig/src/drmmode_display.c	2010-09-16 20:14:25.000000000 +0200
+++ xf86-video-ati-6.13.1/src/drmmode_display.c	2010-09-16 20:18:12.000000000 +0200
@@ -1430,7 +1430,7 @@
 	u = udev_new();
 	if (!u)
 		return;
-	mon = udev_monitor_new_from_netlink(u, "udev");
+	mon = udev_monitor_new_from_netlink(u);
 	if (!mon) {
 		udev_unref(u);
 		return;
diff -Nur xf86-video-ati-6.13.1-orig/src/drmmode_display.h xf86-video-ati-6.13.1/src/drmmode_display.h
--- xf86-video-ati-6.13.1-orig/src/drmmode_display.h	2010-09-16 20:14:25.000000000 +0200
+++ xf86-video-ati-6.13.1/src/drmmode_display.h	2010-09-16 20:18:26.000000000 +0200
@@ -31,6 +31,7 @@
 
 #include "xf86drmMode.h"
 #ifdef HAVE_LIBUDEV
+#define LIBUDEV_I_KNOW_THE_API_IS_SUBJECT_TO_CHANGE
 #include "libudev.h"
 #endif
 
